frame/scheduler/src/tests.rs

```diff
@@ -30,6 +30,7 @@ use sp_runtime::traits::Hash;
 use substrate_test_utils::assert_eq_uvec;
 
 #[test]
+#[docify::export]
```

Comment:
You can try and add a bit of explanation/docs to these tests as well, if suitable.